queries/dune_v2/period_slippage.sql

```diff
@@ -103,12 +99,51 @@ other_transfers as (
       and (t.evt_tx_hash = lower('{{TxHash}}') or '{{TxHash}}' = '0x')
       and (solver_address = lower('{{SolverAddress}}') or '{{SolverAddress}}' = '0x')
 ),
-batch_transfers as (
+eth_transfers as (
+    select
+        bt.tx_hash,
+        from as sender,
+        to as receiver,
+        '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee' as token,
+        cast(value as decimal(38, 0)) as amount_wei,
+        'Native ETH' as transfer_type
```

Comment:
Can we also classify them as AMM IN and AMM OUT? I think they its easier to know whether it was a transfer away from the settlement contract or into the settlement contract

Reply:
Yes sure! Although I suspect that we will only ever see these as `AMM_OUT` -- I suppose both are possible.